composer.json

```diff
@@ -7,7 +7,8 @@
     "require": {
         "php": ">=5.5.9",
         "laravel/framework": "5.1.*",
-        "guzzlehttp/guzzle": "~5.3|~6.0"
+        "guzzlehttp/guzzle": "~5.3|~6.0",
+        "laravelcollective/html": "5.1.*"
```

Comment:
These changes are not related to login form.

This probably happened due to wrong branch creation/merges.